### ISSUE TITLE
Rich text field button options don't have consistent rendering order

### ISSUE THREAD

**Issue by yuvi100** [verbb/formie#1055]
### Describe the bug

The order in which you click the checkboxes for the rich text field options is the order that it renders on the page. The order should remain consistent regardless of the order they're checked in the field settings.

**Render Preview:**
![image](https://user-images.githubusercontent.com/4101371/186306593-0e88ed2a-2bf8-4d39-8583-5c1bbdaac67e.png)

### Steps to reproduce

1. Create 2 Multi-line Text fields.
2. Enable "Use Rich Text Field" option for both.
3. Select the following Rich Text Buttons for the first Multi-line Text fields in this order: Ordered List, Unordered List.
4. Select the following Rich Text Buttons for the second Multi-line Text field in this order: Unordered List, Ordered List.

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.2.1.1

### Plugin version

dev-craft-4 as 2.0.5

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.8](https://github.com/verbb/formie/releases/tag/2.0.8)
